keyboards/45_ats/config.h

```diff
@@ -0,0 +1,52 @@
+ /* 
+ Copyright 2020 Alec Penland
+ Copyright 2020 Garret Gartner
+  
+ This program is free software: you can redistribute it and/or modify 
+ it under the terms of the GNU General Public License as published by 
+ the Free Software Foundation, either version 2 of the License, or 
+ (at your option) any later version. 
+  
+ This program is distributed in the hope that it will be useful, 
+ but WITHOUT ANY WARRANTY; without even the implied warranty of 
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+ GNU General Public License for more details. 
+  
+ You should have received a copy of the GNU General Public License 
+ along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+ */ 
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
```

Comment:
You have a `via` keymap present, but VIA actively discourages use of `0xFEED` as a Vendor ID.

Simple suggestion:

```suggestion
#define VENDOR_ID       0xAB13
```